### PR TITLE
[Components with more than 1 service] LPS-184034 Refactor Editor components from frontend-editor module part I

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://issues.liferay.com/browse/LPS-184034

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:frontend

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions.

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/CKEditorClassicEditor.java [hunk 1]
```diff
@@ -28,47 +25,19 @@
...
-	@Override
-	public String getResourceType() {
-		return PortalWebResourceConstants.RESOURCE_TYPE_EDITOR_CKEDITOR;
-	}
-
 	@Activate
 	protected void activate(Map<String, Object> properties) {
-		_name = (String)properties.get("name");
-	}
+		String name = (String)properties.get("name");
```
Comment (rafaprax) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/CKEditorClassicEditor.java:38]: Hey @MrJohn1911, please inline this statement. Thx

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/CKEditorBalloonEditor.java [hunk 2]
```diff
@@ -27,47 +24,19 @@
...
-	@Override
-	public String getResourceType() {
-		return PortalWebResourceConstants.RESOURCE_TYPE_EDITOR_CKEDITOR;
-	}
-
 	@Activate
 	protected void activate(Map<String, Object> properties) {
-		_name = (String)properties.get("name");
-	}
+		String name = (String)properties.get("name");
```
Comment (rafaprax) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/CKEditorBalloonEditor.java:37]: Hey @MrJohn1911, please inline this statement. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:frontend

**Comment by MrJohn1911** [comment]
ci:stop

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:frontend

**Comment by zhangyan0701** [comment]
ci:test:startup

**Comment by zhangyan0701** [comment]
ci:test:frontend

**Comment by catarineaguiar** [comment]
no regressions found.

**Review by rafaprax — APPROVED** [review]

**Pull request closed without merging**